lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -116,6 +116,14 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
       versioning: `${regexVersioning.id}:^(?<major>\\d+?)\\.(?<minor>\\d+?)\\.(?<patch>\\d+)\\.(?<build>\\d+)$`,
     },
   },
+  cookiecutter: {
+    asdfPluginUrl: 'https://github.com/shawon-crosen/asdf-cookiecutter',
+    config: {
+      datasource: GithubReleasesDatasource.id,
+      packageName: 'cookiecutter/cookiecutter',
+      extractVersion: '^v(?<version>\\S+)',
```

Comment:
```suggestion
```
there is no leading v


https://github.com/cookiecutter/cookiecutter/releases

Reply:
extractVersion removed